README_COPROCESSOR.md

```diff
@@ -29,6 +29,17 @@ across all the components for this test setup.
   Note that this token is saved by docker locally in the clear,
   so it's best to only give it the permissions you need and set the expiration time to 7 days.
 
+### Install Binaries
+
+Make sure you have these binaries installed in your environment:
+
+* [docker](https://docs.docker.com/engine/install)
+* [rust](https://www.rust-lang.org/tools/install)
+* [protobuf](https://grpc.io/docs/protoc-installation)
+* [nodejs](https://nodejs.org/en/download/package-manager)
```

Comment:
for nodejs, does the latest version work in the L1 demo? I think at the moment only v20 works (higher versions won't work), but maybe that's only for development and not related to running the L1 demo.

Reply:
I do not know if later version work, let's precise v20 in this case. Or feel free to test @0xawaz